### PR TITLE
op-guide: Update tidb configuration document about metrics

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @queenypingcap PTAL

### CODE REVIEW

File: op-guide/configuration.md [hunk 1]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: true
 + The value can be (true) or (false). (true) is to enable metrics and pprof. (false) is to disable metrics and pprof.
 
+### --metrics-addr
++ prometheus pushgateway address
++ default: ""
++ Leaves it empty will disable prometheus client from pushing.
```
Comment (siddontang) [inline op-guide/configuration.md]: Leaving

**Comment by siddontang** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM 
@queenypingcap PTAL

File: op-guide/configuration.md [hunk 2]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: true
 + The value can be (true) or (false). (true) is to enable metrics and pprof. (false) is to disable metrics and pprof.
 
+### --metrics-addr
++ prometheus pushgateway address
```
Comment (queenypingcap) [inline op-guide/configuration.md]: the prometheus pushgateway address

File: op-guide/configuration.md [hunk 3]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
...
++ default: ""
++ Leaving it empty will disable prometheus client from pushing.
+
+### --metrics-intervel
++ prometheus client push interval in seconds
```
Comment (queenypingcap) [inline op-guide/configuration.md]: the prometheus client push interval in seconds

File: op-guide/configuration.md [hunk 4]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: true
 + The value can be (true) or (false). (true) is to enable metrics and pprof. (false) is to disable metrics and pprof.
 
+### --metrics-addr
++ prometheus pushgateway address
++ default: ""
++ Leaving it empty will disable prometheus client from pushing.
```
Comment (queenypingcap) [inline op-guide/configuration.md]: Leaving it empty stops the Prometheus client from pushing.

File: op-guide/configuration.md [hunk 5]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
...
+
+### --metrics-intervel
++ prometheus client push interval in seconds
++ default: 0
++ Setting the value to 0 stops prometheus client from pushing.
```
Comment (queenypingcap) [inline op-guide/configuration.md]: prometheus -> Prometheus

**Comment by c4pt0r** [comment]
@shenli ping...

**Comment by shenli** [comment]
@queenypingcap PTAL

File: op-guide/configuration.md [hunk 6]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: true
 + The value can be (true) or (false). (true) is to enable metrics and pprof. (false) is to disable metrics and pprof.
 
+### --metrics-addr
++ Prometheus pushgateway address
```
Comment (queenypingcap) [inline op-guide/configuration.md]: the Prometheus pushgateway address

File: op-guide/configuration.md [hunk 7]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
...
++ default: ""
++ Leaving it empty stops the Prometheus client from pushing.
+
+### --metrics-intervel
++ Prometheus client push interval in seconds
```
Comment (queenypingcap) [inline op-guide/configuration.md]: the Prometheus client push interval in seconds

File: op-guide/configuration.md [hunk 8]
```diff
@@ -64,6 +64,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
...
++ default: 0
++ Setting the value to 0 stops Prometheus client from pushing.
```
Comment (queenypingcap) [inline op-guide/configuration.md]: Prometheus client - > the Prometheus client

**Comment by shenli** [comment]
@queenypingcap PTAL

**Comment by queenypingcap** [comment]
LGTM

**Pull request merged**